### ISSUE TITLE
NTR: Tumor necrosis factor (TNF) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8330]
New term request Tumor necrosis factor (TNF) high in blood (elthat_130413171731): TNFalpha

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000239
- label: Elevated circulating tumor necrosis factor alpha concentration
- parents: [Abnormality of serum cytokine level (HP:0011112)](https://hpo.jax.org/app/browse/term/HP:0011112)
- pmid: PMID:2791581
- def: The concentration of tumor necrosis factor alpha in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8330
- synonyms:
